### DISCUSSION TITLE
calling useBalance

### DISCUSSION THREAD

**Discussion by Seranged** [wevm/wagmi#508] [Q&A] [answered]
sup guys

have been implementing the upg from 0.3.0 to 0.4.7.

we used to be able to use this code to call useBalance as a function - is this not possible anymore? Sorry if i'm misinterpreting the docs. Cheers, Seranged

OLD
  const [{ data: quoteTokenBalance }, getUSDCBalance] = useBalance({
    addressOrName: account,
    token: mockUSDAddress,
  })

NEW
  const getUSDCBalance = useBalance({
    addressOrName: account,
    token: mockUSDAddress,
  })

**Comment by jxom** [comment, marked as answer]
This should work. However, you can use `refetch` in place of `getUSDCBalance`. Check out the full list of return values [here](https://wagmi.sh/docs/hooks/useBalance#return-value).

If you are having an issue, let us know!
